test/tests.cpp

```diff
@@ -1506,3 +1505,54 @@ TEST_CASE_METHOD(QInterfaceTestFixture, "test_swap_reg")
     REQUIRE_FLOAT(qftReg->Prob(0), 0);
     REQUIRE_FLOAT(qftReg->Prob(1), 0);
 }
+#if 0
```

Comment:
But why?

Reply:
This is the 100x Grover's search loop for benchmarking. Ultimately, this will be migrated with other tests into its own suite.

Reply:
This is the 100x Grover's search loop for benchmarking. Ultimately, it will be moved into its own test suite.